rest_server/package.json

```diff
@@ -4,9 +4,6 @@
   "description": "The rest server of openpai marketplace.",
   "author": "Microsoft",
   "license": "MIT",
-  "engines": {
```

Comment:
what is the requirement of node version now?

Reply:
The webportal in marketplace require node 10, but the rest-server works well with node 8.
I just forget to add "node": "^10" back.
Fixed.